<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-30</y>
     <width>611</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px  solid rgba(225,255,255,40);
border-radius: 7px;</string>
   </property>
   <widget class="QLineEdit" name="lineEditLogin_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(225,255,255,0);
border: 0px  solid rgba(225,255,255,0);
color:black;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>401</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(43, 42, 51);</string>
    </property>
    <widget class="QLabel" name="Login">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(225,255,255,100);</string>
     </property>
     <property name="text">
      <string>Логин:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLogin">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(225,255,255,0);
border: 0px  solid rgba(225,255,255,0);</string>
     </property>
    </widget>
    <widget class="QLabel" name="Password">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(225,255,255,100);</string>
     </property>
     <property name="text">
      <string>Пароль:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPass">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(225,255,255,0);
border: 0px  solid rgba(225,255,255,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Gmail">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(225,255,255,100);</string>
     </property>
     <property name="text">
      <string>Gmail:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditGmail">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(225,255,255,0);
border: 0px  solid rgba(225,255,255,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Number">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(225,255,255,100);</string>
     </property>
     <property name="text">
      <string>Number:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNumber">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>230</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(225,255,255,0);
border: 0px  solid rgba(225,255,255,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Password_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(225,255,255,100);</string>
     </property>
     <property name="text">
      <string>Регистрация</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_save">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>260</y>
       <width>79</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: white;
background-color:rgba(255,255,255,0);
}
QPushButton:hover{
background-color:rgba(255,255,255,40)
}
QPushButton:pressed{
background-color:rgba(255,255,255,60)
}
</string>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/icons/icon/login_24dp_E8EAED_FILL0_wght400_GRAD0_opsz24 (1).png</normaloff>:/icons/icon/login_24dp_E8EAED_FILL0_wght400_GRAD0_opsz24 (1).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
